docs/integrations/pydantic.md

```diff
@@ -264,3 +264,52 @@ input_data = UserInput(id='abc', name='Jake')
 # this will run pydantic's validation
 instance = input_data.to_pydantic()
 ```
+
+### Classes with \_\_get_validators\_\_
+
+Pydantic BaseModels may define a custom type with [\_\_get_validators\_\_](https://pydantic-docs.helpmanual.io/usage/types/#classes-with-__get_validators__)
+logic. You will need to add a scalar type and add the mapping to the `scalar_overrides`
```

Comment:
idk what to call it. pydantic calls it "Classes with __get__validators__" so i'm just going to call it that